zilliqa/src/consensus.rs

```diff
@@ -896,26 +1126,36 @@ impl Consensus {
     ) -> Result<()> {
         let Some(new_high_qc_block) = self.block_store.get_block(new_high_qc.block_hash)? else {
             // We don't set high_qc to a qc if we don't have its block.
+            warn!("Recieved potential high QC but didn't have the corresponding block");
             return Ok(());
         };
 
         let new_high_qc_block_view = new_high_qc_block.view();
 
         if self.high_qc.block_hash == Hash::ZERO {
+            // This seems like a potential bug???
```

Comment:
can people not send you a block with hash zero and you end up dumping your highQC?